electron/package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "pack": "electron-builder --dir",
     "dist": "node .electron-vue/build.js && electron-builder",
-    "dev": "node --openssl-legacy-provider .electron-vue/dev-runner.js"
+    "dev": "node --openssl-legacy-provider .electron-vue/dev-runner.js || node .electron-vue/dev-runner.js"
```

Comment:
This breaks it for me. Do we need to support old node?

Reply:
Great, let's get rid of it and see if anyone complains :)

Reply:
Oh, I thought "old" was the other way around :D . Which version of nodejs do you use?

Reply:
Ohhhh, okay.  I guess I could just upgrade my node then :)